tests/test_greedy_monroe.py

```diff
@@ -0,0 +1,19 @@
+from votepy.rules.greedy_monroe import greedy_monroe
+from votepy.ordinal_election import OrdinalElection
+from votepy.rules.k_borda import k_borda
+
+
+def test_greedy_monroe():
+    voting = [
+        [1, 0, 2, 3, 4],
+        [3, 2, 4, 1, 0],
+        [2, 4, 1, 3, 0]
+    ]
+    size_of_committee = 2
+    voting = OrdinalElection(voting)
```

Comment:
You do not need to explicity construct `OrdinalElection`. It is checked implicitly thanks to `@rule` decorator.